threshsign/src/eddsa/EdDSAMultisigSigner.cpp

```diff
@@ -31,5 +33,5 @@ void EdDSAMultisigSigner::signData(const char *hash, int hashLen, char *outSig,
   result.id = id_;
   std::memcpy(outSig, &result, sizeof(SingleEdDSASignature));
 }
-const IShareSecretKey &EdDSAMultisigSigner::getShareSecretKey() const { return getPrivKey(); }
+const IShareSecretKey &EdDSAMultisigSigner::getShareSecretKey() const { return getPrivateKey(); }
```

Comment:
I think I understand why you duplicated the getPrivateKey function in EdDSASigner. You needed to use getPrivKey. please remove the duplication and change only getShareSecretKey.

Reply:
`const PrivateKeyType &getPrivateKey() const;` removed.
`const PrivateKeyType privateKey_;` changed from `private` to `protected`.
Changes in https://github.com/arc-vmware/concord-bft/pull/6/commits/87f4edd0f4f5d72b422cef50f4a8eeeee8b28bc2